.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is ignored here. If this directory is used to store images or other media files that are part of your application's functionality, you might want to reconsider ignoring it. Ensure that any necessary files are not excluded from version control.